src/app/containers/Timestamp/index.test.jsx

```diff
@@ -111,4 +104,60 @@ describe('Timestamp', () => {
       /Updated [0-9]{1,2} \w+ [0-9]{4}/,
     );
   });
+
+  describe('time dependent tests', () => {
```

Comment:
In addition to this can we have 2 more simple tests? Just ones that assert some timestamp in may outputs to something, and one in Jan. Then if we accidentally go to only BST or only UTC, one will fail